.github/workflows/release_dev.yml

```
@@ -35,29 +60,56 @@ jobs:
     needs: build_needed
-    if: needs.build_needed.outputs.needed == 'true'
+    if: needs.build_needed.outputs.needs_build == 'true'
     with:
       yyc: ${{ github.event_name == 'workflow_dispatch' && github.event.inputs.compiler == 'YYC' || github.event_name != 'workflow_dispatch' }}
 
   release:
     name: Release
     runs-on: windows-2022
     needs: gamemaker_build
-    if: needs.build_needed.outputs.needed == 'true'
+    if: needs.build_needed.outputs.needs_build == 'true'
```

actionlint (1.7.4)

property "build_needed" is not defined in object type {gamemaker_build: {outputs: {suffix: string}; result: string}}

(expression)